tests/integration/stepfunctions/v2/services/test_lambda_task_service.py

```diff
@@ -53,3 +54,59 @@ def test_invoke(
             definition,
             exec_input,
         )
+
+    # AWS's stepfuctions documentation seems to incorrectly classify LogType parameters as unsupported.
+    def test_invoke_unsupported_param(
```

Comment:
```suggestion
    # AWS's stepfuctions documentation seems to incorrectly classify LogType parameters as unsupported.
    @pytest.mark.aws_validated
    def test_invoke_unsupported_param(
```

We use the `aws_validated` marker currently as function-level markers to indicate that the test has been verified to pass against AWS